cypress/tests/overview/settings-cluster-general.cy.ts

```diff
@@ -0,0 +1,96 @@
+import { tab } from '../../views/tab';
+
+describe('Test Cluster General settings', () => {
+  before(() => {
+    cy.visit('');
+    cy.wait(15000);
+    cy.switchToVirt();
+    cy.visitVirtPerspectiveOverview();
+    tab.navigateToSettings();
+  });
+
+  describe('Check installed version appears', () => {
+    it('check version', () => {
+      cy.get('[data-test="detail-item-value"]').contains('4');
+      cy.get('[data-test="detail-item-value"]').contains('Up to date');
+    });
+  });
+
+  describe('Test Live migration', () => {
+    it('set live migration limits', () => {
+      cy.contains('General settings').click();
+      cy.contains('Live migration').click();
+      cy.get('input[name="parallelMigrationsPerCluster"]').clear().type('4');
+      cy.get('input[name="parallelOutboundMigrationsPerNode"]').clear().type('1');
+      cy.contains('Set live migration network').click();
+      cy.wait(1000);
+    });
+
+    it('verify live migration limits in HCO', () => {
+      const maxPerCluster = '.spec.liveMigrationConfig.parallelMigrationsPerCluster';
+      const maxPerNode = '.spec.liveMigrationConfig.parallelOutboundMigrationsPerNode';
+      const spec = '.spec.liveMigrationConfig';
+      const cmd = `oc get -n openshift-cnv hyperconverged kubevirt-hyperconverged -o jsonpath='{${spec}}'`;
+      cy.exec(cmd).then((res) => {
+        cy.task('log', res.stdout);
+      });
+      cy.checkHCOSpec(maxPerCluster, '4', true);
+      cy.checkHCOSpec(maxPerNode, '1', true);
+    });
+  });
+
+  describe('Test Memory density', () => {
+    it('ID(CNV-11225) enable memory density', () => {
+      cy.contains('Memory density').click();
+      cy.get('[id="memory-density-feature"]').find('input[type="checkbox"]').check({ force: true });
+    });
+
+    it('verify higherWorkloadDensity in HCO', () => {
+      const percentage = '.spec.higherWorkloadDensity.memoryOvercommitPercentage';
+      cy.checkHCOSpec(percentage, '150', true);
+      cy.wait(10000);
```

Comment:
1. why we need to wait after a check? 
2. where is the source for  `checkHCOSpec()`?

Reply:
1. the wait should be placed after check the option, not here. The purpose is waiting for the backend complete the updating before uncheck the option, remove flaky for the test.
2. add it  now.

Reply:
checkout how @sjd78 implemented active checking in a loop:
https://github.com/konveyor/tackle2-ui/blob/991b02aa1001bf05eb53e868e48b8bb42091e48f/hack/setup-operator.sh#L43
Usage:
```
  echo "Waiting for the Tackle CRD to exist"
  retry_command 10 10 \
    $KUBECTL get customresourcedefinitions.apiextensions.k8s.io tackles.tackle.konveyor.io

```
This way you can provide a big maximal timeout i.e. 10min but at the same time be quick if thing are going smoothly.